### ISSUE TITLE
Filter then select last throws null pointer exception

### ISSUE THREAD

**Issue by bfabry** [nathanmarz/specter#79]
Trying to understand this behaviour:
```
(sp/transform [(sp/filterer odd?) sp/LAST] inc [1 2 3 4])
=> [1 2 4 4]
(sp/select [(sp/filterer odd?) sp/LAST] [1 2 3 4])
java.lang.NullPointerException: 
*e
=>
#error{:cause nil,
       :via [{:type java.lang.NullPointerException, :message nil, :at [clojure.lang.Numbers ops "Numbers.java" 1013]}],
       :trace [[clojure.lang.Numbers ops "Numbers.java" 1013]
               [clojure.lang.Numbers add "Numbers.java" 128]
               [com.rpl.specter$filterer$fn__6601$next_fn__6602 invoke "specter.clj" 229]
               [com.rpl.specter$filterer$fn__6601 invoke "specter.clj" 231]
```

**Comment by JoePython1** [comment]
Not for me:

specterapp.core=> (transform [(filterer odd?) LAST] inc [1 2 3 4])
[1 2 4 4]

**Comment by bfabry** [comment]
Ah! this was fixed in 0.9.3
